Assets/Global/Scripts/Menu/PauseLogic.cs

```diff
@@ -52,7 +54,29 @@ public void QuitGame()
         Menu.SetActive(!Menu.activeSelf);
         // Upgrades.SetActive(!Upgrades.activeSelf);
         Time.timeScale = 1f;
-        SceneManager.LoadScene("Menu");
+        //SceneManager.LoadScene("Menu");
+
+        if (GetCurrentSceneName() == "PARKOUR_1" || GetCurrentSceneName() == "PARKOUR_2" || GetCurrentSceneName() == "PARKOUR_3")
```

Comment:
can this if/else be reversed? 
so logically it'd be:
if (ENTRANCE) => load menu
else => load hub

would let us add levels without worrying about this if statement :)

Reply:
now thinking of it, if we can do the if statement by roomtype would be even better i think as long as it can be done easily.
so:
if (roomtype == entrance) => load menu
else => load hub